src/_locales/en/messages.json

```
@@ -196,6 +196,18 @@
     "message": "Some settings are managed by your system administrator and cannot be changed.",
     "description": "The message, which appears, when settings are pre-defined (as managed options) by administrators."
   },
+  "titleGeneral": {
+    "message": "Add On",
+    "description": "The title for the general settings group."
+  },
+  "titleQrCodeSetting": {
+    "message": "QR Code Settings",
+    "description": "The title for the qr code settings group."
+  },
+  "titleAddonBehavior": {
+    "message": "Addon Behavior",
```

```suggestion
    "message": "Add-on Behavior",
```

https://design.firefox.com/photon/copy/word-list.html#add-on strikes again :wink: